### PR TITLE
custo ckeditor and selectize editors

### PR DESCRIPTION
**Pull request by germanbisurgi**

### CODE REVIEW

File: src/assets/ckeditor.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
+JSONEditor.defaults.editors.ckeditor = JSONEditor.AbstractEditor.extend({
```
Comment (schmunk42) [inline src/assets/ckeditor.js]: Please rename file to `src/assets/editors/ckeditor.js`

File: src/assets/selectizes.js [hunk 2]
```diff
@@ -0,0 +1,180 @@
+JSONEditor.defaults.editors.filefly = JSONEditor.AbstractEditor.extend({
```
Comment (schmunk42) [inline src/assets/selectizes.js]: Please rename file to `src/assets/editors/filefly.js`

**Comment by schmunk42** [comment]
Thank you!

**Comment by germanbisurgi** [comment]
done

**Pull request merged**